src/transformStateWithClones.js

```diff
@@ -65,7 +65,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const result = [];
```

Comment:
I think that it will be better to replace "result" list name by more expressive name, like "statesList"

Reply:
'result' sounds okay